exercises/01-Installing-Plugins/README.md

```diff
@@ -46,4 +48,13 @@ Navigate to http://localhost:8080/restart and click **Yes**, or check the box la
 
 Once Jenkins restarts, you'll be back to the sign in screen and you'll be able to use the new plugin.
 
+**Step 4: Install Other Course-Required Plugins**
+-------------------------------------------
+
+Now that you know how to install plugins, here's some of the other ones we're going to use during the course. Make sure these are installed before finishing the exercise.
+
+**SonarQube Scanner for Jenkins** 
+**GitHub Integration**
+**GitHub Branch Source**
```

Comment:
looks like `GitHub Branch Source` is not the ful name of your plugin

<img width="879" alt="image" src="https://github.com/user-attachments/assets/2d4a48c6-345f-4f5c-bea3-20e70e29ab77">

Reply:
Heh, there's another, separate plugin for github branch source... But this is a good point, I should add in something about unique identifiers.